### ISSUE TITLE
Design Picker v2

### ISSUE THREAD

**Issue by autumnfjeld** [Automattic/wp-calypso#56567]
# Design Picker v2
Continuing work on the Design Picker in Onboarding. Prior GH issues listed below.
This is part of the Blogger Experience Pod work.

![image](https://user-images.githubusercontent.com/5665959/134825087-7862d58c-a08c-4fd4-bf39-046a914b3cb9.png)

## Details
[Figma Desings](https://www.figma.com/file/EuqfSnWpfYx8fgiBlVmbuA/?node-id=2119%3A1107)
**User Facing Specs** (also check for updates  to the Project Thread specs 

- [x] Will be shipped to all English-speaking users
- [x] Will display more themes than v1 (expected 20-30 themes)
- [x] Will include a Categories UI as shown in the image above
- [x] May include the preview feature #56274

## Dev Checklist

- [x] D67279-code Allow the `/template/demo` endpoint to be used with any free theme
- [x] #56528
- [x] #56274
- [x] #56773
  - [x] #56812
  - [x] #56814
  - [x] #56858
  - [x] Update focus rings (something @ollierozdarz is thinking about)
  - [x] #56859
  - [x] #57591
  - [x] Fix theme categories so there are fewer of them, and no duplicates pdgK6S-jl-p2
  - [x] #57986
  - [x] #58151
  - [x] #58268
- [ ] Add Ryu theme to the design picker pdgK6S-mr-p2
- [x] #56832
- [x] #58155

## Related  /  Prior Work
Blogger Onboarding Flow v1 Project Thread: pdgK6S-78-p2
Tech Foundations: Add the design picker in the onboarding flow #55604
Post-checkout Design Picker: Feedback Round 1 #56306
Design Picker: Add design preview #56274

**Comment by autumnfjeld** [comment]
Just adding a little housekeeping comment here:
@alaczek will take the lead on defining the theme categories (as described in the P2 pdgK6S-jl-p2) and organizing that in theme taxonomy system (I'm not familiar with theme category code, so may not have described that very accurately 😄 ).  Once that is done @p-jackson can finish the dev work to plug the new categories into the Design Picker logic and UI.   

![bitmoji](https://sdk.bitmoji.com/render/panel/469d4aea-0aaa-4b89-919e-0fd853b6f0f0-b931236c-c79a-49b5-b0f5-ab26d7e4cd09-v1.png?transparent=1&palette=1&width=56)

**Comment by alaczek** [comment]
Yes! I have been thinking about this past couple of days and I came up with a proposal. 

![image](https://user-images.githubusercontent.com/4550351/141049317-080221cd-2a8c-44e9-8137-12da97c3c754.png)

My guiding principle here was to put a theme in a category that most closely matches the screenshot, to align with customer expectations (so, for example, TwentyTwenty is in the Business category only, even though it can be just as easily used as a blog). This is why most themes are assigned a single category, even though they can be multipurpose. 

I'd love some input on the Recommended category. Do we still need it, if we have the intent capture and can immediately point to the relevant category?

I also identified two emerging categories, and I'd love input on whether or not we should include them.
![image](https://user-images.githubusercontent.com/4550351/141048898-b258b1fa-4a1f-4b3f-bea5-a40dc04fbace.png)


While we're doing this work I'm also considering doing a little clean-up of the more tired designs. I'll look into theme popularity first, to make sure we're not removing something wildly popular. So far these are on my hit list (and not included in any categories above):
* Stow
* Hever
* Morden

cc @ianstewart @jordesign @p-jackson

**Comment by alaczek** [comment]
> While we're doing this work I'm also considering doing a little clean-up of the more tired designs.

I looked at the data, and interestingly Hever, Morden, Exford, and Shawburn consistently show up at the top, so I'm thinking we should keep them for now (all in the Business category), and I'll come back to look at the data a month from now (after we add categorization) to see If they remain popular. 

If they do, we should look into replacing them with newer, better-quality variations that improve upon the original.

**Comment by alaczek** [comment]
We chatted about this in our pod sync, and this is where we landed:
* Recommended category is confusing - because in the theme showcase it is something like an umbrella for ALL the themes we have here. **We're getting rid of it in favor of pointing customer to the category that matches their stated intent.**
* We'll be adding an ALL category instead of Featured. It will be added in code - not as an actual category.
* For the themes in the design picker - we’ve resolved that the themes included there will ONLY have the `theme-subject` terms for which we want it to appear in the design picker. 
* All the Link in Bio themes should also appear in the Single Page category.
* Not removing any themes for now.
* Not adding Culture and Restaurant for now - the offering there is too weak.

Next step: I'll go over the themes in the showcase and assign them theme subjects, which will automatically be reflected in the design step. Once that's done I think we can close this issue.
